testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/ServerRestartRequiredTestCase.java

```diff
@@ -144,29 +134,6 @@ public void stopServers() throws Exception {
     }
 
     @Test
-    public void testSocketBinding() throws Exception {
-        final DomainClient client = domainMasterLifecycleUtil.getDomainClient();
-
-        // add a custom socket binding, referencing a non-existent system property
-        final ModelNode socketBinding = new ModelNode();
-        socketBinding.add(SOCKET_BINDING_GROUP, "reload-sockets");
-        socketBinding.add(SOCKET_BINDING, "my-custom-binding");
-
-        final ModelNode socketBindingAdd = new ModelNode();
-        socketBindingAdd.get(OP).set(ADD);
-        socketBindingAdd.get(OP_ADDR).set(socketBinding);
-        socketBindingAdd.get(PORT).set(new ValueExpression("${my.custom.socket}"));
-
-        // Don't rollback server failures, rather mark them as restart-required
-        socketBindingAdd.get(OPERATION_HEADERS).get(ROLLOUT_PLAN)
-                .get(IN_SERIES).add().get(CONCURRENT_GROUPS).get("reload-test-group").get(MAX_FAILURE_PERCENTAGE).set(100);
-
-        executeOperation(socketBindingAdd, client);
-
-        checkRestartOneAndTwo(client);
-    }
```

Comment:
This seems like a useful test, but needs different checks when the servers are started.